src/main/webapp/scripts/keywords.js

```diff
@@ -15,3 +15,34 @@ $().ready(function() {
     $('#form-title *').text('Copy Your Tweet Here');
   });
 });
+
+/** 
+ * Checks whether the form inputs are valid before submitting said form.
+ */
+function validateForm(event) {
+  if ($('#blob-input').hasClass('hide')) {
+    // Check the text input
+    if ($('#text-input').val() == null || $('#text-input').val() === "") {
+      alert('Your input cannot be empty!'); 
+      event.preventDefault();
+      return false;
+    } else {}
+  } else {  // Check blob input
+    const blobInput = document.getElementById('blob-input');
```

Comment:
Why are you using vanilla javascript to get this element, and not jQuery like in the rest of the script?

Reply:
I didn't learn about the jquery way to do it until just now, so I will be converting it in the next commit.